access/src/main/webapp/WEB-INF/jsp/common/header.jsp

```diff
@@ -20,133 +20,84 @@
-			
-			<ul id="mainmenu">
-				<c:set var="referUrl">
-					<c:choose>
-						<c:when test="${not empty param.refer}">
-							${param.refer}
-						</c:when>
-						<c:otherwise>
-							${currentAbsoluteUrl}
-						</c:otherwise>
-					</c:choose>
-				</c:set>
-			
-				<c:forEach items="${headerMenuSettings.menuRoot.subMenus}" var="menuEntry">
-					<li>
-						<c:set var="activeClass">
-							<c:if test="${requestScope.menuId == menuEntry.key}">
-								active
-							</c:if>
-						</c:set>
-						<c:set var="menuTarget">
-							<c:choose>
-								<c:when test="${not empty menuEntry.value.target}">target="${menuEntry.value.target}"</c:when>
-								<c:otherwise></c:otherwise>
-							</c:choose>
-						</c:set>
-						<c:if test="${not empty menuEntry.value.url}">
-							<c:url var="menuUrl" value="${menuEntry.value.url}">
-								<c:if test="${menuEntry.value.includeReferer}">
-									<c:param name="refer" value="${referUrl}"/>
-								</c:if>
-							</c:url>
-							<a href="<c:out value='${menuUrl}' />" class="${activeClass}" ${menuTarget}>${menuEntry.value.label}</a>
-						</c:if>
-						<c:if test="${not empty menuEntry.value.subMenus}">
-							<ul class="submenu">
-								<c:forEach items="${menuEntry.value.subMenus}" var="subEntry">
-									<li>
-										<c:set var="menuTarget">
-											<c:choose>
-												<c:when test="${not empty subEntry.value.target}">target="${subEntry.value.target}"</c:when>
-												<c:otherwise></c:otherwise>
-											</c:choose>
-										</c:set>
-										<c:url var="menuUrl" value="${subEntry.value.url}">
-											<c:if test="${subEntry.value.includeReferer}">
-												<c:param name="refer" value="${referUrl}"/>
-											</c:if>
-										</c:url>
-										<a href="<c:out value='${menuUrl}' />" ${menuTarget}>${subEntry.value.label}</a>
-									</li>
-								</c:forEach>
-							</ul>
-						</c:if>
-					</li>
-				</c:forEach>
-			</ul>
-		</div>
-	</div>
-	<div id="searchoptions">
-		
-		<div class="contentarea">
-		<div id="searchoptions-bottom">
-			<div id="advancedsearch">
-			<a href="advancedSearch">Advanced Search</a>
-			</div>
-				<form class="right clear_on_submit_without_focus" method="get" action="basicSearch" id="hsearch_form">
-						<input name="queryType" type="hidden" value="${searchSettings.searchFieldParams['DEFAULT_INDEX']}">
-						<div id="hsearch_inputwrap">
-						   <input name="query" type="text" id="hsearch_text" placeholder="Search all collections">
-						   <input type="submit" value="Go" id="hsearch_submit">
-						</div>
-				 </form>
-		</div>
-	</div>
-</div>
-		</div>
-	</div>
-</div>
-</div>
-</div>
\ No newline at end of file
+<header class="">
+    <div class="logo-row">
+        <div class="logo container">
+            <a href="">
+                <img src="static/images/university-libraries-logo.png" alt="University Libraries Logo">
```

Comment:
I think this is the wrong url, it is failing to load. I believe the correct one is `static/front/university-libraries-logo.png`